shared/src/test/scala/coop/rchain/shared/GeneratorUtils.scala

```diff
@@ -13,17 +13,21 @@ object GeneratorUtils {
   def distinctListOf[T: Arbitrary] =
     distinctListOfGen(arbitrary[T])(_ == _)
 
+  def limitedDistinctList[T: Arbitrary](maxSize: Int) =
```

Comment:
This shouldn't be needed. It should be enough to supply the `size` parameter to the `forAll` construct